send-notification.ts

```diff
@@ -0,0 +1,36 @@
+/** A notification to be sent. */
+interface Notification {
+    /** The user ID of the recipient. */
+    userId: string;
+    /** The subject of the notification. */
+    subject: string;
+    /** The content of the notification. */
+    content: { [key: string]: any };
+    /** The code of the notification. */
+    code: number;
+    /** The sender ID of the notification. */
+    senderId?: string | null;
```

Comment:
does senderId refer to userId as well? I suppose can name it senderUserId if thats the case